### PR TITLE
add en-US/snippets/2017/apr2017.lang

### PR DESCRIPTION
**Pull request by peiying2**
per bug 1345221

### CODE REVIEW

File: en-US/snippets/2017/apr2017.lang [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## NOTE: Reference bug 1345221; Deadline: 21 March, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotion for Firefox Focus through snippet and email
+## NOTE: There is no character length limit
+
+
+# Campaign: Firefox desktop snippet
+;Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
```
Comment (flodolo) [inline en-US/snippets/2017/apr2017.lang]: No double spaces after a period.

File: en-US/snippets/2017/apr2017.lang [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Campaign: Email headline for Firefox Focus on iOS and Android
+;Seriously easy privacy.
+Seriously easy privacy.
+
+
+# Campaign: Email body text for Firefox Focus on iOS and Android
+# URL for iOS: https://itunes.apple.com/app/firefox-focus-the-privacy-browser/id1055677337?mt=8
+# No link for Android
+;Block unwanted ads and trackers by default with Firefox Focus. Install it today.
+Block unwanted ads and trackers by default with Firefox Focus. Install it today.
```
Comment (flodolo) [inline en-US/snippets/2017/apr2017.lang]: Do we know if the entire string is going to become a link or just the last part? If it's the latter, we should mark it with `<a>Install it today.</a>`

Why no link for Android? Do you have more info besides the spreadsheet?

File: en-US/snippets/2017/apr2017.lang [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## NOTE: Campaign: cross platform promotion for Firefox Focus through snippet and email
+## NOTE: There is no character length limit
+
+
+# Campaign: Firefox desktop snippet
+;Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
+Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
+
+
+# Campaign: Email subject line for Firefox Focus on iOS and Android
```
Comment (flodolo) [inline en-US/snippets/2017/apr2017.lang]: You shouldn't put "Campaign" in all these comments, it makes sense only in the first one

File: en-US/snippets/2017/apr2017.lang (hunk 1 above)
Comment (peiying2) [inline en-US/snippets/2017/apr2017.lang]: I guess I am not the only. Copied and pasted... fixed.

File: en-US/snippets/2017/apr2017.lang (hunk 3 above)
Comment (peiying2) [inline en-US/snippets/2017/apr2017.lang]: ok, i will remove them.

File: en-US/snippets/2017/apr2017.lang (hunk 2 above)
Comment (peiying2) [inline en-US/snippets/2017/apr2017.lang]: Mike and exchanged emails multiple times to get this straightened out. He favors Slack for future conversation!  I can summarize the conversation and put it in the bug.  

I am not clear exactly on where the link should be, I would  think it is the latter.

File: en-US/snippets/2017/apr2017.lang (hunk 1 above)
Comment (peiying2) [inline en-US/snippets/2017/apr2017.lang]: also no length constraint, Mike confirmed on this.

**Pull request merged**